NanaZip.UI.Classic/SevenZip/CPP/7zip/UI/GUI/CompressDialog.cpp

```diff
@@ -1366,7 +1366,9 @@ bool CCompressDialog::OnCommand(int code, int itemID, LPARAM lParam)
       {
         Get_FormatOptions().ResetForLevelChange();
 
-        SetMethod();
+        // **************** NanaZip Modification Start ****************
+        SetMethod(GetMethodID());
```

Comment:
Maybe you need to comment out the original code. It's OK if you don't want to modify that because this part will be finally moved into NanaZip.Universal. But I hope you can do that.

Reply:
Do you want it before or after the modification mark?